### ISSUE TITLE
Mac OS download?

### ISSUE THREAD

**Issue by EnsarTahic** [anhbinhvodanh/Dell-5547-Hackintosh#3]
Hi, well I have an issue. I do not know where to get MacOS as I do not own, and do not know anyone who own macos or macbook.. Can you link me or help me get anything?

**Comment by EnsarTahic** [comment]
I couldn't find any working one, do you suggest anything?

**Comment by EnsarTahic** [comment]
is this able to be used for Dell Inspiron 5567 as I get this error and can't pass it 
![image](https://user-images.githubusercontent.com/38700588/40867767-4d58045e-6607-11e8-8c71-4cf396917eae.jpg)

**Comment by kauefontes** [comment]
@EnsairTahic do you get It working with 5567?

**Comment by EnsarTahic** [comment]
I haven’t actually, and I just left it. So I never got it working sadly.

**Comment by kauefontes** [comment]
Okay... I'll keep trying and If i have any success i'll post in my Github... Do you still have the 5567? Thanks 4 the quick response =D

**Comment by EnsarTahic** [comment]
Hello, sorry for again not responding that fast. And yes I still have it.

**Comment by anhbinhvodanh** [comment]
Sorry all, but I think have many differences between 2 models 5547 and 5567.
So I am not sure it will work on your laptop.

**Comment by kauefontes** [comment]
Yeah... I'm working with others threads, If i get Amy success i'll update here and post my Clover Folders/files into my Github =D

**Comment by anhbinhvodanh** [comment]
Ok. That's sound good.
